### PR TITLE
Add redirecUri to samples

### PR DESCRIPTION
**Pull request by andrewvanbeektech**
these samples are missing the redirectUri

## Description:
- **What's changed?** 
- **Is this PR related to a Monolith release?** 

### Resolves:

* [OKTA-XXXXXX](https://oktainc.atlassian.net/browse/OKTA-XXXXXX)

### CODE REVIEW

File: packages/@okta/vuepress-site/code/javascript/okta_sign-in_widget/index.md [hunk 1]
```diff
@@ -139,6 +139,7 @@ In this case, you would like to use the Widget to sign in to a simple web page a
     <script type="text/javascript">
       const oktaSignIn = new OktaSignIn({
         baseUrl: "https://${yourOktaDomain}",
+        redirectUri: '{{redirectUri configured in OIDC app}}',
```
Comment (vijetmahabaleshwar-okta) [inline packages/@okta/vuepress-site/code/javascript/okta_sign-in_widget/index.md:142]: Can we change this to `redirectUri configured in **your** OIDC app` to be more specific?

**Review by vijetmahabaleshwar-okta — APPROVED** [review]
Approved with a minor comment

File: packages/@okta/vuepress-site/code/javascript/okta_sign-in_widget/index.md (hunk 1 above)
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/code/javascript/okta_sign-in_widget/index.md:142]:
```suggestion
        redirectUri: '{{https://{yourAppRedirectUri} configured in your OIDC app}}',
```

**Review by susanharper-okta — COMMENTED** [review]
one change

**Pull request merged**